modules/gdscript/gdscript.cpp

```diff
@@ -928,6 +933,12 @@ Variant GDScript::callp(const StringName &p_method, const Variant **p_args, int
 		top = top->_base;
 	}
 
+	MethodBind *method = ClassDB::get_method(native->get_name(), p_method);
+	if (method && method->is_static()) {
+		// Native static method.
+		return method->call(nullptr, p_args, p_argcount, r_error);
+	}
```

Comment:
Please add a test, if possible.

Reply:
added a native test class with a bound static method and tests